Resources/Locale/en-US/chat/highlights.ftl

```diff
@@ -0,0 +1,57 @@
+# Command
+highlights-captain = Captain, "Cap", Command
+highlights-head-of-personnel = Head Of Personnel, "HoP", Service, Command
+highlights-chief-engineer = Chief Engineer, "CE", Engineering, Engineer, "Engi", Command
+highlights-chief-medical-officer = Chief Medical Officer, "CMO", MedBay, "Med", Command
+highlights-head-of-security = Head of Security, "HoS", Security, "Sec", Command
+highlights-quartermaster = Quartermaster, "QM", Cargo, Command
+highlights-research-director = Research Director, "RD", Science, "Sci", Command
+
+# Security
+highlights-detective = Detective, "Det", Security, "Sec"
+highlights-security-cadet = Security Cadet, Cadet, Security, "Sec"
+highlights-security-officer = Security Officer, Secoff, Officer, Security, "Sec"
+highlights-warden = Warden, "Ward", Security, "Sec"
+
+# Cargo
+highlights-cargo-technician = Cargo Technician, Cargo Tech, Cargo
+highlights-salvage-specialist = Salvage Specialist, Salvager, Salvage, "Salv", Miner
+
+# Engineering
+highlights-atmospheric-technician = Atmospheric Technician, Atmos tech, Atmospheric, Atmos
+highlights-station-engineer = Station Engineer, Engineering, Engineer, "Engi"
+highlights-technical-assistant = Technical Assistant, Tech Assistant, Engineering, Engineer, "Engi"
+
+# Medical
+highlights-chemist = Chemist, Chemistry, MedBay
+highlights-medical-doctor = Medical Doctor, Doctor, "Doc", MedBay, "Med"
+highlights-medical-intern = Medical Intern, Intern, MedBay, "Med"
+highlights-paramedic = Paramedic, "Para", MedBay, "Med"
+
+# Science
+highlights-scientist = Scientist, Science, "Sci"
+highlights-research-assistant = Research Assistant, Science, "Sci"
+
+# Civilian
+highlights-bartender = Bartender, Barkeeper, Barkeep, "Bar"
+highlights-botanist = Botanist, Botany, Hydroponics
+highlights-chaplain = Chaplain, "Chap", Chapel
+highlights-chef = Chef, "Cook", Kitchen
+highlights-clown = Clown, Jester
+highlights-janitor = Janitor, "Jani"
+highlights-lawyer = Lawyer, Attorney
+highlights-librarian = Librarian, Library
+highlights-mime = Mime
+highlights-passenger = Passenger
+highlights-service-worker = Service Worker
+
+# Station-specific
+highlights-boxer = Boxer
+highlights-reporter = Reporter, Journalist
+highlights-zookeeper = Zookeeper
+highlights-psychologist = Psychologist, Psychology
+
+# Silicon
+highlights-personal-ai = Personal AI, "pAI"
+highlights-cyborg = Cyborg, Borg
+highlights-station-ai = Station AI, "AI"
```

Comment:
```suggestion
highlights-cyborg = Cyborg, Sillicon, Borg
highlights-station-ai = Station AI, Sillicon, "AI", "sAI"
```

Borgs could get called sillicon, and i sometimes see people call AI as sAI

Reply:
> Borgs could get called sillicon, and i sometimes see people call AI as sAI

Thank you for the reviews! Regarding this suggestion, would "silicon" also work for addressing borgs?

Reply:
> Thank you for the reviews! Regarding this suggestion, would "silicon" also work for addressing borgs?

Oops yes my bad, i forgot its with a single L